### DISCUSSION TITLE
Welcome to ChatGPT-Siri Discussions!

### DISCUSSION THREAD

**Discussion by Yue-Yang** [Yue-Yang/ChatGPT-Siri#4] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by zoomy942** [comment]
Hey there.  I'm trying they Siri Shortcut and I love the idea.  I have followed the instructions and pasted the API key and all that.  I'm getting an error - Could not evaluate the key path.  In '', no value was found for the dictionary key 'choices'.

**Reply by whateverone2** [reply]
I had the same problem.  You have to have a paid account with openai.  The API has its own pricing, which can be found at https://openai.com/pricing

**Reply by vizualspeed** [reply]
Thx, I had the same problem, .. guess it's time to pay

**Comment by Yue-Yang** [comment]
check #30 ，for this question
